static-site/content/docs/07-visualisation/02-map-interpretation.md

```diff
@@ -0,0 +1,98 @@
+
+![](./img/labelled-movement.png)
+_Figure 3.
+Left: Cartoon of how the tree determines the construction of transmission lines.
+Right: Example of transmission lines showing [between country movement of Ebola](https://nextstrain.org/ebola)._
+
+**Line thickness.**
+Each rendered arc is actually a collection of individual lines of identical thickness (as above, each line identified by a branch in the tree indicating a change in the value of the current geographic resolution).
+Lines which look "thicker" therefore represent more observed cases of movement between the two locations (circles).
+For instance, in figure 3 the arc labelled by "1" is thicker than that labelled "3", indicating that in the [Ebola dataset](https://nextstrain.org/ebola) there are more observed cases of movement between Guinea and Sierra Leone than there are between Liberia and Guinea.
+
+
+**Directionality** is conveyed by the [orientation of the arc](https://en.wikipedia.org/wiki/Curve_orientation).
+A transmission line is a positively oriented curve when traveling from source to origin (parent node to child node on the tree). 
+In figure 3, line 1 is positively curved when moving from Guinea to Sierra Leone and therefore indicates the inferred viral movements from Guinea to Sierra Leone; line 2 represents movements from Sierra Leone to Guinea.
+
+
+**Line color** 
+It represents the currently selected "color-by" -- the metadata displayed in the legend -- of each transmission.
+If we examine the cartoon example in figure 3, we see that the parent and child nodes have a different location, which we will use to draw a line on the map from location A to location B. If the current color-by is "date", then the line will be colored blue, which corresponds to the metadata at the origin of the transmission.
+As transmission arcs potentially consist of a number of lines, we can have multiple different colors within a transmission arc between locations. 
+Looking at the transmission from Guinea to Sierra Leone (labelled "1" in Figure 3), we can see that there are some transmissions from earlier in the outbreak (blue colors) as well as a number of transmissions from later on (orange colors).
+
+If the selected color-by is not present at the origin of an individual transmission, then the line will be colored grey.
+It's important to realise that the line color is not related to the geographic resolution, which determines the presence and layout of the lines.
+
+
+**Uncertainty** is always inherent in the reconstructions from which the transmission data is drawn.
+Often this uncertainty is encoded in the dataset, and we use this to reduce the saturation of branch colors in the phylogenetic tree.
+Currently we don't convey this uncertainty in the map -- we simply use whatever values the dataset defines for the geographic resolution in question. 
+
+
+## Animation
+
+If we click the "Play" button (Figure 1, label "4") we can watch the outbreak unfold before our eyes. 
```

Comment:
Small thing but I like this notation and am in favor of adopting it consistently throughout the doc (and others with labelled figures:
> (Figure 1, label "4")

Reply:
Agreed!